test/release/examples/hello.chpl

```diff
@@ -1,2 +1,2 @@
 // Simple hello world
-writeln("Hello, world!");    // print 'Hello, world!' to the console
+writeln("Hello, world");    // print 'Hello, world!' to the console
```

Comment:
@klakho0400 please remove that ! in the comment as well.

Reply:
i did not mean to change it

Reply:
@ben-albrecht  can you revert it back

Reply:
@klakho0400 no worries.

Reply:
@klakho0400 - you'll need to remove this change before I can merge.